src/base/ApplicationLoadBalancer.ts

```diff
@@ -1,7 +1,13 @@
-import { Resource } from 'cdktf';
+import { Resource, TerraformVariable, VariableType, Fn } from 'cdktf';
 import { vpc, elb, datasources, s3, iam } from '@cdktf/provider-aws';
 import { Construct } from 'constructs';
 
+const ALB_ACCOUNT_ID_MAP = {
+  us_east_1: '127311923021',
+  us_west_1: '027434742980',
+  us_west_2: '797873946194',
+};
```

Comment:
ugh, assuming these can't be derived in any way. ☹️ could these be in a config?

Reply:
You should be able to get the id from terraform https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity

Reply:
@bkochendorfer i had that originally. Unfortunately. This is the elb Account ID.  https://docs.aws.amazon.com/elasticloadbalancing/latest/application/enable-access-logging.html#enable-access-logs

Reply:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/elb_service_account

Reply:
@bkochendorfer you are my hero. I didnt even think terraform would have this.